File: Timensit_Backend/TimensitApp/src/app/views/pages/Timensit/danh-muc/che-do-uu-dai/Model/che-do-uu-dai.model.ts
import { stringify } from 'querystring';
import { BaseModel } from "../../../../../../core/_base/crud";

export class chedouudaiModel extends BaseModel {
    Id: number;
    CheDoUuDai: string;
    MoTa: string;
    Locked: boolean;
    Priority: number;
    CreatedBy: number;
    CreatedDate: string;
    UpdatedBy: number;
    UpdatedDate: string;

	clear() {
        this.CheDoUuDai = '';
		this.MoTa = '';
		this.Locked = true;
        this.Priority = 1;
        this.CreatedBy = 0;
        this.CreatedDate = '';
        this.UpdatedBy = 0;
        this.UpdatedDate = '';

        this.Id = 0;
	}
}

File: NCC_API/Models/DanhMuc/CheDoUuDaiModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Timensit_API.Models
{

    public partial class CheDoUuDaiModel
    {
        public int Id { get; set; }
        public string CheDoUuDai { get; set; }
        public string MoTa { get; set; }
        public bool Locked { get; set; } = false;
        public int Priority { get; set; } = 0;
    }
}
